src/sgame/Entities.cpp

```diff
@@ -151,6 +152,7 @@ bool Entities::KnockbackRadiusDamage(Entity& entity, float amount, float range,
 		if (!G_IsVisible(entity.oldEnt, other.oldEnt, MASK_SOLID)) return;
 
 		glm::vec3 knockbackDir = VEC2GLM( other.oldEnt->s.origin ) - VEC2GLM( entity.oldEnt->s.origin );
+		normalize_warn( knockbackDir );
```

Comment:
This isn't actually normalized here so I guess calling this is just an accident.